examples/Training/python/Raw_Data_Access.py

```diff
@@ -78,6 +78,11 @@
 for i, p in enumerate(planes):
     print "plane zct:", zct_list[i], " min:", p.min(), " max:", p.max()
 
+# Retrieve a histogram
+# ====================
+# Get a 256 bin histogram for channel 0 and plane z=0/t=0:
+hist = pixels.getHistogram(0, 0, 0, 256)
```

Comment:
maybe you had good reason to put the getHistogram method on the PixelsWrapper, I wanted to mention that there is a getHistogram on the ImageWrapper already. Tried this locally:

```
hist = image.getHistogram([0], 256, False, 0, 0)
>>> hist
{0L: [1L, 0L, 0L, 0L, 0L, 0L, 0L, 0L, 0L, 6L, 0L, 0L, 9L, 0L, 51L, 0L, 0L, 131L, 0L, 0L, 430L, 0L, 0L, 967L, 0L, 0L, 2194L, 0L, 4056L, 0L, 0L, 7470L, 0L, 0L, 11290L, 0L, 0L, 16620L, 0L, 21019L, 0L, 0L, 25216L, 0L, 0L, 26803L, 0L, 0L, 27057L, 0L, 0L, 23321L, 0L, 20227L, 0L, 0L, 16262L, 0L, 0L, 12444L, 0L, 0L, 8395L, 0L, 0L, 5897L, 0L, 3813L, 0L, 0L, 2644L, 0L, 0L, 1864L, 0L, 0L, 1311L, 0L, 1016L, 0L, 0L, 771L, 0L, 0L, 673L, 0L, 0L, 562L, 0L, 0L, 520L, 0L, 438L, 0L, 0L, 369L, 0L, 0L, 369L, 0L, 0L, 343L, 0L, 363L, 0L, 0L, 337L, 0L, 0L, 346L, 0L, 0L, 361L, 0L, 0L, 333L, 0L, 310L, 0L, 0L, 349L, 0L, 0L, 384L, 0L, 0L, 381L, 0L, 0L, 351L, 0L, 366L, 0L, 0L, 351L, 0L, 0L, 411L, 0L, 0L, 366L, 0L, 447L, 0L, 0L, 425L, 0L, 0L, 450L, 0L, 0L, 432L, 0L, 0L, 515L, 0L, 542L, 0L, 0L, 510L, 0L, 0L, 561L, 0L, 0L, 505L, 0L, 561L, 0L, 0L, 527L, 0L, 0L, 520L, 0L, 0L, 567L, 0L, 0L, 551L, 0L, 471L, 0L, 0L, 448L, 0L, 0L, 472L, 0L, 0L, 447L, 0L, 0L, 432L, 0L, 401L, 0L, 0L, 400L, 0L, 0L, 320L, 0L, 0L, 318L, 0L, 268L, 0L, 0L, 274L, 0L, 0L, 249L, 0L, 0L, 203L, 0L, 0L, 184L, 0L, 132L, 0L, 0L, 120L, 0L, 0L, 92L, 0L, 0L, 53L, 0L, 50L, 0L, 0L, 48L, 0L, 0L, 30L, 0L, 0L, 19L, 0L, 0L, 10L, 0L, 10L, 0L, 0L, 3L, 0L, 0L, 7L, 0L, 0L, 2L, 0L]}

```